libs/service-portal/signature-collection/src/screens/hooks.ts

```diff
@@ -80,48 +74,42 @@ export const useGetSignedList = () => {
     loading: loadingSignedList,
     refetch: refetchSignedList,
   } = useQuery<SignedList>(GetSignedList)
-
   const signedList =
     (getSignedList?.signatureCollectionSignedList as SignatureCollectionList) ??
-    {}
+    null
   return { signedList, loadingSignedList, refetchSignedList }
 }
 
-export const useGetOwnerLists = () => {
-  const { data: getOwnerLists, loading: loadingLists } =
-    useQuery<OwnerLists>(GetOwnerLists)
-
-  const ownerLists =
-    (getOwnerLists?.signatureCollectionListsByOwner as SignatureCollectionList[]) ??
-    []
-  return { ownerLists, loadingLists }
-}
-
-export const useGetListsBySigneeArea = (area: string) => {
+export const useGetListsForUser = () => {
   const {
-    data: getListsBySigneeArea,
-    loading: loadingAreaLists,
-    refetch: refetchListsBySigneeArea,
-  } = useQuery<ListsByArea>(GetListsBySigneeArea, {
-    variables: {
-      input: {
-        areaId: area,
-      },
-    },
-  })
+    data: getListsForUser,
+    loading: loadingUserLists,
+    refetch: refetchListsForUser,
+  } = useQuery<ListsForUser>(GetListsForUser)
 
-  const listsBySigneeArea =
-    (getListsBySigneeArea?.signatureCollectionListsByArea as SignatureCollectionList[]) ??
+  const listsForUser =
+    (getListsForUser?.signatureCollectionListsForUser as SignatureCollectionList[]) ??
     []
-  return { listsBySigneeArea, loadingAreaLists, refetchListsBySigneeArea }
+  return { listsForUser, loadingUserLists, refetchListsForUser }
 }
 
 export const useIsOwner = () => {
-  const { data: getIsOwner, loading: loadingIsOwner } =
-    useQuery<IsOwner>(GetIsOwner)
+  const {
+    data: getIsOwner,
+    loading: loadingIsOwner,
+    refetch: refetchIsOwner,
+  } = useQuery<IsOwner>(GetIsOwner)
 
   const isOwner =
     (getIsOwner?.signatureCollectionIsOwner as SignatureCollectionSuccess) ??
     false
-  return { isOwner, loadingIsOwner }
+  return { isOwner, loadingIsOwner, refetchIsOwner }
+}
+
+export const useCancelCollection = () => {
+  const [cancelCollection, { loading: loadingCancelCollection }] =
+    useMutation<SignatureCollectionSuccess>(CancelCollection, {
+      variables: { input: { id: '1000027' } },
```

Comment:
I don't know what this hard-coded id is.
Is it possible to leave behind a comment explaining the id value?

Reply:
Updated so it sends in the currentCollection from the lists returned as a param.